### DISCUSSION TITLE
Is there a way to get the position of the scaffold in the parent molecule?

### DISCUSSION THREAD

**Discussion by Vencent-Won** [rdkit/rdkit#5640] [Q&A]
for example, there is a molecule "O=\[N+\]\(\[O-\]\)c1ccc2c\(ccc3ccccc32\)c1", and its scaffold is "c1ccc2c\(c1\)ccc1ccccc12"; 
Can I get the map of atoms of "c1ccc2c\(c1\)ccc1ccccc12" in the "O=\[N+\]\(\[O-\]\)c1ccc2c\(ccc3ccccc32\)c1"?
Or, get the groups connected atoms ids?

**Comment by Vencent-Won** [comment]
Sorry, I didn't search carefully before. And then I found ptosco's reply to group point labeling, the method is really smart, thanks

**Reply by ShangChien** [reply]
can you provide a link of Ptosco's reply？thank you!

**Reply by Vencent-Won** [reply]
You can use the property function to add the atom-id property to achieve motivation.
```python
mol = Chem.MolFromSmiles('************')
for a in mol.GetAtoms():
    a.SetIntProp("__origIdx", a.GetIdx())
```

**Reply by ShangChien** [reply]
get it, thankyou!
